src/ui/views/session/overlay/Overlay.css

```diff
@@ -37,3 +37,7 @@
   height: 100%;
   object-fit: cover;
 }
+
+.Overlay__world-preview--blur {
+  filter: blur(16px);
```

Comment:
When you blur an image like this, you get black bands on the sides because of how the blur algorithm works. You can add something like `transform: scale(1.1);` to avoid this.